ecommerce/extensions/api/v2/tests/views/test_publication.py

```diff
@@ -523,5 +525,21 @@ def test_mobile_seats_update(self, _, __):
             response = self.client.put(self.update_path, json.dumps(updated_data), JSON_CONTENT_TYPE)
 
             self.assertEqual(response.status_code, 200)
-            self.assert_course_saved(self.course_id, expected=updated_data,
-                                     enrollment_code_count=1, test_mobile_seats=True)
+            self.assert_course_saved(self.course_id, expected=updated_data, enrollment_code_count=1)
+
+    @mock.patch('ecommerce.extensions.api.serializers.create_ios_product')
+    def test_ios_seat_created(self, mock_create_ios_product):
+        """Verify that a Course and associated mobile products can be updated and published."""
+        self.create_course_and_seats()
+        updated_data = self.generate_update_payload()
+
+        with mock.patch.object(LMSPublisher, 'publish') as mock_publish:
```

Comment:
can you remove mock from here and add something like this
```suggestion
        @mock.patch.object(LMSPublisher, 'publish', return_value=None)
def test_course_update_creates_ios_seat_when_feature_enabled(self, mock_publish, mock_create_ios_product):
```